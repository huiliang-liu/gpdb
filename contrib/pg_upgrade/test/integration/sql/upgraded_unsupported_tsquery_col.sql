ALTER TABLE tsquery_tbl ALTER COLUMN b TYPE TSQUERY USING b::tsquery;
SELECT * FROM tsquery_tbl;
SELECT * FROM tsquery_tbl WHERE b @> 'New';
