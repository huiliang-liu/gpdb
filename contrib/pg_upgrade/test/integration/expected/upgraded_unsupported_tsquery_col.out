ALTER TABLE tsquery_tbl ALTER COLUMN b TYPE TSQUERY USING b::tsquery;
ALTER
SELECT * FROM tsquery_tbl;
 a | b              
---+----------------
 a | 'New' & 'York' 
 a | 'New' & 'York' 
 a | 'New' & 'York' 
 a | 'New' & 'York' 
(4 rows)
SELECT * FROM tsquery_tbl WHERE b @> 'New';
 a | b              
---+----------------
 a | 'New' & 'York' 
 a | 'New' & 'York' 
 a | 'New' & 'York' 
 a | 'New' & 'York' 
(4 rows)
